package.json

```diff
@@ -151,5 +152,10 @@
   "volta": {
     "node": "16.13.0",
     "npm": "8.1.1"
+  },
+  "dependencies": {
+    "@actions/core": "^1.9.1",
+    "@actions/github": "^5.0.3",
+    "@vercel/ncc": "^0.34.0"
```

Comment:
I think these should be able to move to dev-deps, since we only need them when compiling the action code.

Reply:
Is there anything stopping us from putting these in a `package.json` file in `.github/workflows/actions/auto-request-reviews`? Is there a requirement for them to be in the root level `package.json`? I wonder if it'd be better to keep these separate from the rest of the dependencies 🤔

Reply:
Good call, was able to make that change